java/ql/lib/ext/java.lang.reflect.model.yml

```diff
@@ -0,0 +1,9 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: neutralModel
+    data:
+      - ["java.lang.reflect", "Constructor", "newInstance", "(Object[])", "manual"]
+      - ["java.lang.reflect", "Field", "get", "(Object)", "manual"]
+      - ["java.lang.reflect", "Method", "getName", "()", "manual"]
+      - ["java.lang.reflect", "Method", "invoke", "(Object,Object[])", "manual"]
```

Comment:
> Should any of the java.lang.reflect models be summaries instead of neutral?

As above, this _seems_ useful for a specific query (unsafe reflection) but we can always add those as specific taint steps for the query, or switch them to summaries here when necessary.

Reply:
Okay, sounds good, I’ll leave these and the other reflection-related methods above as neutrals for now.